pkg/generic/binarypb_test/generic_test.go

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2024 CloudWeGo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package test
+
+import (
+	"context"
+	"log"
+	"testing"
+	"time"
+
+	"github.com/cloudwego/kitex/client/callopt"
+	"github.com/cloudwego/kitex/internal/test"
+	"github.com/cloudwego/kitex/pkg/generic/binarypb_test/kitex_gen/base"
+	"github.com/cloudwego/kitex/pkg/generic/binarypb_test/kitex_gen/example2"
+	"google.golang.org/protobuf/proto"
+)
+
+func TestRun(t *testing.T) {
+	t.Run("TestBasicServiceExampleMethod", TestBasicServiceExampleMethod)
+	t.Run("TestExample2ExampleMethod", TestExample2ExampleMethod)
```

Comment:
It would be great if we could also test cases where an error is thrown and when the server is a normal server.